spec/support/unit/record_validating_confirmation_builder.rb

```diff
@@ -6,6 +6,7 @@ class RecordValidatingConfirmationBuilder
 
     def initialize(options)
       @options = options
+      @data_type = options[:data_type].nil? ? :string : options[:data_type]
```

Comment:
Let's do this:

* I believe that I call this option `column_type` in other places, so let's rename the option to that.
* Let's move this to a method under `private`.
* Let's use `fetch` on Hash to handle the use-value-if-key-given-otherwise-use-another-value logic.

So, all together, that method would look like:

``` ruby
private

def column_type
  options.fetch(:column_type, :string)
end
```